.travis.yml

```diff
@@ -1,12 +1,12 @@
-# Lets actually use our tests! :smile:
+sudo: false
 branches:
   only:
     - master
 
-sudo: required
 language: ruby
 cache: bundler
 dist: trusty
+rvm: 2.3.3
```

Comment:
ChefDK 1.4 uses ruby 2.3; ChefDK 2 will use ruby 2.4.1. What you're showing there is the version of rubygems, not ruby :)